rest/class.wp-super-cache-rest-get-settings.php

```diff
@@ -198,7 +197,7 @@ protected function get_is_lock_down_enabled() {
 	 * @return int
 	 */
 	protected function get_post_count() {
-		$posts_count = wp_count_posts();
+		$count = wpsc_post_count();
```

Comment:
Maybe, we can use `$count = $this->get_post_count();` instead `$count = wpsc_post_count();`